chain/network/src/utils.rs

```diff
@@ -5,9 +5,12 @@ use std::{hash::Hash, net::IpAddr};
 
 use crate::types::{BlockedPorts, PatternAddr};
 
-pub fn blacklist_from_vec(blacklist: &Vec<String>) -> HashMap<IpAddr, BlockedPorts> {
+pub fn blacklist_from_vec<T>(blacklist: T) -> HashMap<IpAddr, BlockedPorts>
```

Comment:
Probably the name of this function should also be changed.

Reply:
👍 
https://github.com/nearprotocol/nearcore/pull/2634/commits/4ae0be76ff62be60c58eb9b60e367c7d27eb52f9